lib/bas/process/discord/implementation.rb

```diff
@@ -11,24 +11,36 @@ module Discord
     # for sending messages to Discord.
     #
     class Implementation < Base
+      attr_reader :webhook, :name
+
+      # Initializes the processor with essential configuration parameters.
```

Comment:
🤔 what do you think? 
```suggestion
      # Initializes the Process with essential configuration parameters.
```